pennylane_lightning/core/src/simulators/lightning_qubit/gates/cpu_kernels/GateImplementationsLM.hpp

```diff
@@ -454,7 +456,7 @@ class GateImplementationsLM : public PauliGenerator<GateImplementationsLM> {
                          const std::vector<bool> &controlled_values,
                          const std::vector<std::size_t> &wires,
                          FuncT core_function) {
-        constexpr std::size_t one{1};
+        static constexpr std::size_t one{1};
```

Comment:
does it really make any difference in performance? :thinking:

Reply:
No, but it is a good idea rather than declaring it every time.

Reply:
I always come back to this comment every time I ask myself the same thing: https://stackoverflow.com/a/13867690

Reply:
fair! :t-rex:

Reply:
fair! :)

Reply:
okay :)